task-ballerina/build.gradle

```
@@ -99,16 +101,20 @@ task copyStdlibs(type: Copy) {
 task updateTomlVersions {
     inputs.dir file(project.projectDir)
     doLast {
+        def stdlibDependentTimeVersion = project.stdlibTimeVersion.split("-")[0]
```

I don't know from where you referred when adding this build file, but it seems outdated. Please refer to a new build.gradle file.

```suggestion
        def stdlibDependentTimeVersion = project.stdlibTimeVersion.replace("SNAPSHOT", "")
```